recipe/meta.yaml

```diff
@@ -24,16 +24,24 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - toml
+    - tomli >=1.1.0
 
 test:
   requires:
     - pip
+    - coverage ==7.0.5
+    - pint
+    - pytest ==7.2.1
+    - pytest-cov ==4.0.0
   imports:
     - vulture
+  source_files:
+    - vulture
```

Comment:
Think adding `vulture` to `source_files` would mean we might pick that up in the path instead of the one we installed. Would it be possible to test without this? Or is there some small set of files we need from the source?

Also if this is needed because of `doctest`s or something else, we can look at adding `--pyargs vulture` to the `pytest` command below

Reply:
That's an interesting point about the doctests. For my other projects, we do run the doctests, hence why it's needed. I wasn't aware of `--pyargs`.

I can revert this in the meantime.